.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: node_js
+node_js:
+  - "node"
+notifications:
+  email: false
+env:
```

Comment:
And while you look into that, is it possible that we cache node_modules here? I'm not sure (correct me if I'm wrong), I believe it makes it run more efficiently in the case of a large project.

Reply:
@darasimiolaifa  as we are moving on it will be modified, i just needed to test whether Travis was running.

Reply:
No problem. Just felt to point it out. Good job nonetheless.